src/init-project/files/testing/jest.config.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  preset: 'jest-preset-angular',
+  setupFilesAfterEnv: [
+    '<rootDir>/setup-jest.ts'
+  ],
+  setupFiles: [
+    '<rootDir>/setup-jest-dependencies.ts'
+  ],
+  testPathIgnorePatterns: [
+    '<rootDir>/node_modules/',
+    '<rootDir>/dist/',
+    '<rootDir>/src/test.ts',
+    '<rootDir>/global-mocks.ts'
+  ],
+  globals: {
+    'ts-jest': {
+      tsConfig: '<rootDir>/tsconfig.spec.json',
+      stringifyContentPathRegex: '\\.html$',
+      astTransformers: [
+        '<rootDir>/node_modules/jest-preset-angular/build/StripStylesTransformer',
+        '<rootDir>/node_modules/jest-preset-angular/build/InlineFilesTransformer'
+      ],
+      
```

Comment:
@astorozhevsky 

Please remove this blank line and the trailing comma on line 22.